web/client/actions/mapInfo.js

```diff
@@ -193,6 +195,20 @@ function featureInfoClick(point, layer) {
     };
 }
 
+function highlightFeature(enabled) {
+    return {
+        type: HIGHLIGHT_FEATURE,
+        enabled
+    };
+}
+
+function changeIndex(index) {
```

Comment:
Changes the index of what?

Reply:
Index of the current response shown in the feature info panel in the list of valid responses (sub-group of responses). 
The property in the state is mapInfo.index. Should I document the action and the reducer's parts?